content/04.results.md

```diff
@@ -9,5 +9,36 @@ place the grid of aurocs here for discriminator model
 ## Random Noise Generative Model
 place the results of random label function experiment
 
-## Reconstructing Hetionet
-place figure of number of new edges that can be added to hetionet as well as edges we can reconstruct using this method
+
+### Reconstructing Hetionet
+Once the discriminiative model has been calibrated, we compared our model to a baseline model [@tag:cocoscore] (Figure {@fig:cocoscore_comparison}) and  estimated the number of edges that can be added/recalled from Hetionet v1 (Figure Figure {@fig:hetionet_reconstruction}).
```

Comment:
```suggestion
Once the discriminiative model has been calibrated, we compared our model to a baseline model [@tag:cocoscore] (Figure {@fig:cocoscore_comparison}) and  estimated the number of edges that can be added/recalled from Hetionet v1 (Figure {@fig:hetionet_reconstruction}).
```

Reply:
Also, I'm not sure why the comparison and adding edges are stuck together as a combined section. Is there something that links them?